package/samplers/cmamae/example.py

```diff
@@ -0,0 +1,50 @@
+import optuna
+import optunahub
+from optuna.study import StudyDirection
+
+from sampler import CmaMaeSampler
+
+# TODO: Replace above import with this.
+#  module = optunahub.load_module("samplers/pyribs")
+#  PyribsSampler = module.PyribsSampler
+
+
+def objective(trial: optuna.trial.Trial) -> float:
+    x = trial.suggest_float("x", -10, 10)
+    y = trial.suggest_float("y", -10, 10)
+    return -(x**2 + y**2) + 2, x, y
+
+
+if __name__ == "__main__":
+    sampler = CmaMaeSampler(
+        param_names=["x", "y"],
+        archive_dims=[20, 20],
+        archive_ranges=[(-10, 10), (-10, 10)],
+        archive_learning_rate=0.1,
+        archive_threshold_min=-10,
+        n_emitters=1,
+        emitter_x0={
+            "x": 5,
+            "y": 5
+        },
+        emitter_sigma0=0.1,
+        emitter_batch_size=5,
+    )
+    study = optuna.create_study(
+        sampler=sampler,
+        directions=[
+            # pyribs maximizes objectives.
+            StudyDirection.MAXIMIZE,
+            # The remaining values are measures, which do not have an
+            # optimization direction.
+            # TODO: Currently, using StudyDirection.NOT_SET is not allowed as
```

Comment:
What direction should I use here? It seems Optuna only accommodates minimize and maximize. Quality diversity inherently involves a "no direction" optimization, hence the "diversity" aspect. Does it work to just use MINIMIZE as a placeholder?

Reply:
Usually, you would like to determine the sign of `values` for samplers and `directions` gives you the information for this.
If your sampler doesn't use this information, you can simply ignore this parameter, meaning that the default will be used.

Reply:
I see. The reason I looked into this is that the objective function for quality diversity algorithms like CMA-MAE needs to return multiple values, i.e., the objective and the measures. When I did not specify `directions`, I got an error that the objective function was only supposed to return one value.